packages/@aws-cdk/aws-apigatewayv2/lib/http/authorizer.ts

```diff
@@ -78,7 +121,7 @@ export interface HttpAuthorizerAttributes {
   /**
    * Type of authorizer
    */
-  readonly authorizerType: HttpAuthorizerType
+  readonly authorizerType: HttpRouteAuthorizationType
```

Comment:
There's two problems I see here. (a) `HttpRouteAuthorizationType` is not exported but `HttpAuthorizerAttributes` is, and (b) jsii does not support union types. 
By unsupported, this will get converted either into `java.lang.String` or `java.lang.Object` in the case of java.

It's better to just change this to 'string' and add a validation in the constructor that it's one of the valid types.

Reply:
Forgot about that jsii thing 🤦🏾‍♂️  should be implemented now